### PR TITLE
changed makefile to use 'all' architecture regardless

### PR DESCRIPTION
**Pull request by BoopBoopBeepBoop**
since it doesn't appear to affect the output, just the arch flag

### CODE REVIEW

**Comment by BoopBoopBeepBoop** [comment]
@jharveysmith @mbautin I don't have write access, but here is the one change that I made.

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ PKG_VER := $(shell cd chronos && \
 PKG_REL := 0.1.$(shell date -u +'%Y%m%d%H%M%S')
 
 FPM_OPTS := -s dir -n chronos -v $(PKG_VER) \
-	--architecture native \
+	-a all \
```
Comment (mbautin) [inline Makefile:15]: This should still be `--architecture` for readability.

**Comment by mbautin** [comment]
We should create a "csd" branch in https://github.com/clearstorydata/chronos-pkg off of master and push this there instead. Also submit this change to the main chronos-pkg project.

**Comment by BoopBoopBeepBoop** [comment]
ok, I can't write to our chronos-pkg repo though, so someone else has to make the branch

**Pull request closed without merging**